### PR TITLE
Selection sort in c

### LINKED ISSUE #893: Selection sort in c
<hr>

## Description 📜

To show selection sort technique in c language via documentation.
<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

### PR DESCRIPTION
**Pull request by AditiRout**
<hr>

## Description 📜

Added a file to show implementation of Selection sort in c language 
Fixes #893

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] C/CPP

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Comment by gunjan-g** [comment]
@AditiRout great work, just put this file in correct folder

**Review by gunjan-g — CHANGES_REQUESTED** [review]
@AditiRout great work.
Just put this file in correct folder

**Comment by AditiRout** [comment]
Thank u @gunjan-g maam, may I know which folder should I put it in or should I create a separate folder and then put my file?

**Comment by gunjan-g** [comment]
> Thank u @gunjan-g maam, may I know which folder should I put it in or should I create a separate folder and then put my file?

Create a subfolder in data structures folder named sorting and put your files there

File: C_CPP/readme.md [hunk 1]
```diff
@@ -1,45 +1,4 @@
...
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Operator%20Overloading">Operator Overloading </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Advance%20C%2B%2B%20OOPs%20programming">Advance C++ OOPs programming </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Templates%20%26%20Libraries">Templates & Libraries </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Exception%20Handling">Exception Handling </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/File%20Handling%20%26%20IO%20Management">File Handling & IO Management </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Number%20System%20i.e%20Binary%2C%20Octal%2C%20Decimal%2C%20HexaDecimal%20etc">Number System i.e Binary, Octal, Decimal, HexaDecimal etc </a>
-### <a href="https://github.com/girlscript/winter-of-contributing/tree/C_CPP/C_CPP/Data%20Structures">Data Structures </a>
+-[Selection Sort in C](https://github.com/AditiRout/winter-of-contributing/blob/turtle/C_CPP/Selection%20sort%20in%20c.md)
```
Comment (Suvraneel) [inline C_CPP/readme.md:2]: Not this readme
Put this as readme inside the sorting folder

**Review by Suvraneel — CHANGES_REQUESTED** [review]

File: C_CPP/readme.md (hunk 1 above)
Comment (Suvraneel) [inline C_CPP/readme.md:2]: Not this file, put this as readme inside the sorting folder

**Review by Suvraneel — CHANGES_REQUESTED** [review]

**Review by Suvraneel — CHANGES_REQUESTED** [review]
@AditiRout 
You've done the correct thing now but deleted the original readme...

**Comment by AditiRout** [comment]
So what should I do now?

**Comment by AditiRout** [comment]
@Suvraneel sir is there any other way to get the original readme

**Comment by Suvraneel** [comment]
@AditiRout 
image

**Comment by AditiRout** [comment]
@Suvraneel sir i tried from the fetch and merge from the branch turtle but the option is not available it shows that its already updated.

**Comment by Suvraneel** [comment]
@AditiRout 
found your issue
Do 1 thing

Fetch upstream >  Comapre > then fetch from girlscript/C_CPP

**Comment by AditiRout** [comment]
@Suvraneel sir could you please verify my work?

**Comment by Suvraneel** [comment]
@AditiRout  Technically correct & perfect
I hope you have now learnt how to resolve git conflicts/merges :octocat:

One last thing, regarding content :
Just add Time & Space Complexity Analysis.. That will make the documentation complete

**Comment by Suvraneel** [comment]
@AditiRout  Awesome :heart: Keep up!

**Comment by AditiRout** [comment]
Thank you @Suvraneel sir

**Pull request merged**